CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
 
-project(RigelEngine)
+project(RigelEngine VERSION 0.7.1)
```

Comment:
used the version you last tagged, but please let me know how you'd prefer to do the versioning @lethal-guitar

Reply:
Works for me! 👍🏻

Reply:
Works for me 👍🏻 

(Re-added comment)